pkg/addons/data/DaemonSet.apps/kube-service-catalog/apiserver.yaml

```diff
@@ -1,10 +1,11 @@
 apiVersion: apps/v1
-kind: Deployment
+kind: DaemonSet
```

Comment:
is moving apiserver to a daemonset desired?

Reply:
This is upstream change. Need to discuss with the team what is our goal. Do we want to keep our own implementation or align with upstream?